app/controllers/admin/routes.py

```diff
@@ -174,3 +193,33 @@ def addParticipants():
 
     return render_template('addParticipants.html',
                             title="Add Participants")
+
+@admin_bp.route('/<username>/courseProposals', methods=['GET'])
+def createTable(username, unauthorized=False):
+    if unauthorized:
+        flash("Unauthorized to perform this action", 'warning')
+    try:
+        if authorized(username):
+            courseDict = getProposalData(g.current_user)
+            return render_template("/admin/courseProposals.html",
+                                    instructor = g.current_user,
+                                    courseDict = courseDict,
+                                    username = username)
+        else:
+            flash("Unauthorized to view page", 'warning')
+            return redirect(url_for('main.events'))
+    except Exception as e:
+        print('Error while creating table: ', e)
+        return "", 500
+
+@admin_bp.route('/<username>/courseProposals/<courseID>/withdraw/', methods = ['POST'])
```

Comment:
Why are there two different course withdrawal URLs and two different controller methods for doing it?

Reply:
removed. whoops, mis-pulling from branch went under the radar